modules/fledgeForGpt.js

```diff
@@ -48,5 +47,50 @@ export function addComponentAuctionHook(next, bidRequest, componentAuctionConfig
     logWarn(MODULE, `unable to register component auction config for: ${adUnitCode} x ${seller}.`);
   }
 
-  next(bidRequest, componentAuctionConfig);
+  next(adUnitCode, componentAuctionConfig);
 }
+
+export function markBidsForFledge(next, bidderRequests) {
```

Comment:
This function enables FLEDGE for bidderRequests, should the name be `markBidderRequestsForFledge` or just `markForFledge` ?